third_party/forked/gotestsum/junitxml/report.go

```diff
@@ -33,6 +33,8 @@ type JUnitTestCase struct {
 	Time        string            `xml:"time,attr"`
 	SkipMessage *JUnitSkipMessage `xml:"skipped,omitempty"`
 	Failure     *JUnitFailure     `xml:"failure,omitempty"`
+	SystemOut   string            `xml:"system-out,omitempty"`
+	SystemErr   string            `xml:"system-err,omitempty"`
```

Comment:
please educate me,  these values are part of the junit spec, right https://github.com/testmoapp/junitxml?tab=readme-ov-file#complete-junit-xml-example ?

Reply:
I don't know why I  was assuming that klog was logging to stderr